tools/build_pytorch_libs.sh

```diff
@@ -274,42 +274,39 @@ function build_caffe2() {
 
   if [[ $RERUN_CMAKE -eq 1 ]] || [ ! -f CMakeCache.txt ]; then
       ${CMAKE_VERSION} $BASE_DIR \
-		       ${CMAKE_GENERATOR} \
-		       -DCMAKE_INSTALL_MESSAGE="LAZY" \
-		       -DPYTHON_EXECUTABLE=$PYTORCH_PYTHON \
-		       -DBUILDING_WITH_TORCH_LIBS=ON \
-		       -DCMAKE_BUILD_TYPE=$BUILD_TYPE \
-		       -DBUILD_TORCH=$BUILD_TORCH \
-		       -DBUILD_PYTHON=$BUILD_PYTHON \
-		       -DBUILD_SHARED_LIBS=$BUILD_SHARED_LIBS \
-		       -DBUILD_BINARY=$BUILD_BINARY \
-		       -DBUILD_TEST=$BUILD_TEST \
-		       -DINSTALL_TEST=$INSTALL_TEST \
-		       -DBUILD_CAFFE2_OPS=$BUILD_CAFFE2_OPS \
-		       -DONNX_NAMESPACE=$ONNX_NAMESPACE \
-		       -DUSE_CUDA=$USE_CUDA \
-		       -DCAFFE2_STATIC_LINK_CUDA=$CAFFE2_STATIC_LINK_CUDA \
-		       -DUSE_ROCM=$USE_ROCM \
-		       -DUSE_NNPACK=$USE_NNPACK \
-		       -DUSE_LEVELDB=$USE_LEVELDB \
-		       -DUSE_LMDB=$USE_LMDB \
-		       -DUSE_OPENCV=$USE_OPENCV \
-		       -DUSE_GLOG=OFF \
-		       -DUSE_GFLAGS=OFF \
-		       -DUSE_SYSTEM_EIGEN_INSTALL=OFF \
-		       -DCUDNN_INCLUDE_DIR=$CUDNN_INCLUDE_DIR \
-		       -DCUDNN_LIB_DIR=$CUDNN_LIB_DIR \
-		       -DCUDNN_LIBRARY=$CUDNN_LIBRARY \
-		       -DUSE_MKLDNN=$USE_MKLDNN \
-		       -DMKLDNN_INCLUDE_DIR=$MKLDNN_INCLUDE_DIR \
-		       -DMKLDNN_LIB_DIR=$MKLDNN_LIB_DIR \
-		       -DMKLDNN_LIBRARY=$MKLDNN_LIBRARY \
-		       -DCMAKE_INSTALL_PREFIX="$INSTALL_DIR" \
-		       -DCMAKE_EXPORT_COMPILE_COMMANDS=1 \
-		       -DCMAKE_C_FLAGS="$USER_CFLAGS" \
-		       -DCMAKE_CXX_FLAGS="$USER_CFLAGS" \
```

Comment:
can you undo the tab / whitespace changes, and only make changes to this file that you did. I believe you just added `-DUSE_MKLDNN=$USE_MKLDNN ` to this whole file

Reply:
Sure. Will fix it soon.